function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
For future documentation, let's write a small `quickstart.md` with some basic examples from `usage.ipynb`.

Reply:
added as a section in Readme, including also Streamlit app running script